.github/workflows/tests.yml

```
@@ -15,13 +15,17 @@ concurrency:
 env:
   FORCE_COLOR: 2
 
+defaults:
+  run:
+    shell: bash -elo pipefail {0}
```

Should this be (like in https://github.com/cylc/cylc-flow/pull/6655)?
```suggestion
    shell: bash -c "exec $CONDA_PREFIX/bin/bash -elo pipefail {0}"
```

---

`$CONDA_PREFIX/bin/bash` points at the `bash` executable installed by Micromamba.

These Python tests don't have such shell requirements so this PR sticks with using the system installed Bash.